indico/modules/categories/templates/display/base.html

```diff
@@ -86,6 +91,19 @@ <h1 class="category-title {{ category_title_classes }}">
     </div>
 </div>
 <script>
+    $('#navigate-button').on('click', function(evt) {
+        evt.preventDefault();
+        $('<div>').categorynavigator({
+            openInDialog: true,
+            category: {{ category.id }},
+            actionButtonText: $T.gettext('Navigate to'),
+            dialogTitle: $T.gettext('Navigate to a category'),
+            onAction: function(category) {
+                var url = build_url(Indico.Urls.Categories.display, {category_id: category.id});
+                window.location.href = url;
```

Comment:
`window.location.href = build_url(...)` isn't much longer ;)

(if you want to keep it, use `const` instead of `var`)

Reply:
Changed.